cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,42 @@
 /// <reference types="cypress" />
-
 describe('Sign In page', () => {
+  beforeEach(() => {
+  cy.visit('/')
+});
```

Comment:
There is only one test - you may move beforeEach action into the test body and delete beforeEach